python/chainer_trt/model_retriever.py

```diff
@@ -39,6 +39,11 @@ def chainer_ver():
 
 class RetainHook(chainer.FunctionHook):
     def forward_postprocess(self, function, in_data):
+        dummy_out = function.forward(in_data)
+        function.retain_outputs(list(range(len(dummy_out))))
```

Comment:
c.f. https://github.com/belltailjp/chainer/tree/6250-out-data-to-forward-postprocess